misc/pod.test.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Please move the Dockerfile to the `test/testdata` directory, refer to https://github.com/golang-standards/project-layout/tree/master/test.

Reply:
Fixed